.circleci/config.yml

```diff
@@ -55,6 +55,18 @@ jobs:
         paths:
         - ./architect
 
+  e2eTestBasic:
+    environment:
+      TEST_DIR: "integration/test/basic"
+      TEST_NAME: "basic"
+    <<: *e2eTest
+
+  e2eTestMigration:
+    environment:
+      TEST_DIR: "integration/test/migration"
+      TEST_NAME: "migration"
+    <<: *e2eTest
```

Comment:
Lets just enable the basic test.

The migration test is no longer needed. It was for migrating clusters to use this helm chart.